build/plugins/anchors.html

```diff
@@ -10,7 +10,8 @@
 <script type="module">
   // which types of elements to add anchors next to, in "document.querySelector"
   // format
-  const typesQuery = 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]';
+  const typesQuery =
+    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';
```

Comment:
Is `div[id^="fig:"]`the workaround for double table ids (on the div and table) mentioned at https://github.com/manubot/rootstock/pull/425#issuecomment-853395226? It works by only matching div ids that start with `tbl:` as opposed to all elements?

Just to confirm, it is okay for us to proceed despite the duplicate ids, because this PR handles it?

Reply:
Opened https://github.com/tomduck/pandoc-tablenos/issues/31

Reply:
> Is div[id^="fig:"]the workaround for double table ids (on the div and table) mentioned at #425 (comment)? It works by only matching div ids that start with tbl: as opposed to all elements?

That's correct. I've essentially made the queries more specific. As long as pandoc/table-nos is consistent and never outputs a table without the div wrapper, this should be fine.

Looked at the issue, 👍 FWIW id's being unique isn't just a suggestion, it's a requirement of the spec and can cause unexpected behavior. It shouldn't cause anything to crash though.

Reply:
Got it.

> As long as pandoc/table-nos is consistent and never outputs a table without the div wrapper, this should be fine.

Based on the early discussion in the issue, it seems like the most likely fix is that pandoc-tablenos stops adding the div wrapper for tables since the id is now set in `<figure>`.

So if we upgrade pandoc-tablenos, we might have to tweak this, but that's okay. Let's wait to see what solution pandoc-tablenos adopts and merge this now.

Reply:
Oh if that’s the case we’ll need to re-add the plug-in to wrap tables in divs for scrolling purposes 😞